### PR TITLE
Navbar feature

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:
Added feature test for navbar
Added navbar
#### What I have learned working on this feature: 
* To create small pull requests often rather then a few large ones. 
* Set up partials
* Create partials content (navbar) 

#### Screenshots:
skjermbilde 2018-02-28 kl 22 29 34

### CODE REVIEW

**Comment by kmthorsnes** [comment]
removed [WIP] due to navbar in place and tests passing

File: app/views/partials/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class='nav-item active'>
+            <%= link_to 'Euphony', root_path, class: 'nav-link' %>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Concert tickets</a>
```
Comment (SophieSjoberg) [inline app/views/partials/_navbar.html.erb]: Try using this placeholder:
```
<%= link_to 'Concert tickets', '#', class: 'nav-link' %>
```
It's always better to use the helpers Rails gives us

Comment (SophieSjoberg) [inline app/views/partials/_navbar.html.erb]: Try using this placeholder:
```
<%= link_to 'Concert tickets', '#', class: 'nav-link' %>
```
It's always better to use the helpers Rails gives us

Comment (kmthorsnes) [reply app/views/partials/_navbar.html.erb]: Thank you. 
Adjusted for this in latest commit.

**Review by tochman — APPROVED** [review]
Looks good

**Pull request merged**